stdlib/SparseArrays/src/linalg.jl

```diff
@@ -1295,6 +1295,11 @@ function opnormestinv(A::AbstractSparseMatrixCSC{T}, t::Integer = min(2,maximum(
 end
 
 ## kron
+@inline Base.@propagate_inbounds function kron!(C::SparseMatrixCSC, A::SparseMatrixCSC, B::SparseMatrixCSC)
+    mA, nA = size(A); mB, nB = size(B); mC, nC = size(C)
+    @boundscheck if nC == nA * nB && mC == mA * mB
+        throw(DimensionMismatch("A has size ($mA, $nA), B has size ($mB, $nB), C has size ($mC, $nC)"))
+    end
```

Comment:
CI failures point to a missing 
```suggestion
    end
end
```
here